README.md

```diff
@@ -43,7 +43,8 @@ binary releases.
 
 | Release                                                                | Maintained | Compatible Cilium Versions |
 |------------------------------------------------------------------------|------------|----------------------------|
-| [v0.15.23](https://github.com/cilium/cilium-cli/releases/tag/v0.15.23) | Yes        | Cilium 1.14 and newer      |
+| [v0.15.23](https://github.com/cilium/cilium-cli/releases/tag/v0.15.23) | Yes        | Cilium 1.15 and newer      |
+| [v0.15.22](https://github.com/cilium/cilium-cli/releases/tag/v0.15.22) | Yes        | Cilium 1.14                |
 | [v0.14.8](https://github.com/cilium/cilium-cli/releases/tag/v0.14.8)   | Yes        | Cilium 1.13                |
 
```

Comment:
could you maybe add a sentence or two here and describe why `>=v0.15.23` is incompatible with cilium v1.14?

Reply:
here is the reason https://github.com/cilium/cilium-cli/pull/2285#issuecomment-1926303693.

Do you want to mention in the docs as well ?

Reply:
yeah i think it makes sense to add it in README.md

Reply:
ah right, I have added a small footnote for v0.15.22.